simple-loadflow/src/main/java/com/powsybl/loadflow/simple/ac/ClosedBranchReactiveFlowEquationTerm.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.loadflow.simple.ac;
+
+import com.powsybl.iidm.network.Branch;
+import com.powsybl.iidm.network.Bus;
+import com.powsybl.loadflow.simple.equations.EquationContext;
+import com.powsybl.loadflow.simple.equations.EquationType;
+import com.powsybl.loadflow.simple.equations.Variable;
+import com.powsybl.loadflow.simple.network.BranchCharacteristics;
+import com.powsybl.loadflow.simple.network.NetworkContext;
+
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class ClosedBranchReactiveFlowEquationTerm extends AbstractClosedBranchAcFlowEquationTerm {
+
+    public ClosedBranchReactiveFlowEquationTerm(ClosedBranchAcContext branchContext, Branch.Side side, Bus bus1, Bus bus2,
+                                                NetworkContext networkContext, EquationContext equationContext) {
+        super(branchContext, side, bus1, bus2, EquationType.BUS_Q, networkContext, equationContext);
+    }
+
+    @Override
+    public double eval(double[] x) {
+        Objects.requireNonNull(x);
+        double v1 = x[branchContext.getV1Var().getColumn()];
+        double v2 = x[branchContext.getV2Var().getColumn()];
+        double ph1 = x[branchContext.getPh1Var().getColumn()];
+        double ph2 = x[branchContext.getPh2Var().getColumn()];
+        switch (side) {
+            case ONE:
+                return branchContext.q1(v1, v2, ph1, ph2);
+            case TWO:
+                return branchContext.q2(v1, v2, ph1, ph2);
+            default:
+                throw new IllegalStateException("Unknown side: " + side);
+        }
+    }
+
+    private double dq1(Variable variable, double v1, double v2, double ph1, double ph2) {
```

Comment:
Same as for active power: have lower level methods ?
Maybe the dispatching could be common with active power.